packages/amplication-client/src/Project/ArchitectureConsole/ModelOrganizer.scss

```diff
@@ -20,11 +20,7 @@ $addon-title-line-height: 36px;
   }
 
   &__confirmationDialog {
-    display: flex;
-    padding: 24px;
-    flex-direction: column;
-    align-items: center;
-    gap: 24px;
+    justify-content: center;
```

Comment:
why do we need this? can you share a screenshot?

Reply:
<img width="932" alt="image" src="https://github.com/amplication/amplication/assets/97830649/2171c0c3-69d8-42aa-9480-b21b6d5ba9e8">

Reply:
Without it: 
<img width="892" alt="image" src="https://github.com/amplication/amplication/assets/97830649/2e9b891c-1b56-4ad8-a2b4-0f64d6a08732">